.github/actions/hugo-build-versioned-action/action.yaml

```diff
@@ -35,17 +36,34 @@ runs:
       run: |
         python -m pip install --upgrade pip
         pip install -r scripts/script-requirements.txt
-    - name: Generate Versioned Documentation
+    - uses: actions/setup-go@v4
+      with:
+        go-version: '>=1.20.1'
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Install dependencies
+      working-directory: ${{ inputs.working-directory }}
+      run: npm ci
+      shell: bash
```

Comment:
Is this necessary? If I am not mistaken this should be resolved by peaceiris/actions-hugo@v2 action.

Reply:
Or more precisely, based on the documentation node setup should follow after peaceiris/actions-hugo@v2

Reply:
We can try to omit this, but I copied it from our main documentation and wanted to be extra sure, that it would run.